__tests__/comments.test.js

```diff
@@ -0,0 +1,56 @@
+const app = require('../app')
+const data = require('../db/data/test-data/index')
+const db = require('../db/connection')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+
+require("pg").defaults.parseInt8 = true;
+
+beforeEach( () => {
+    return seed(data)
+})
+
+afterAll( () => {
+    return db.end()
+})
+
+
+describe("GET/api/articles/:article_id/comments", () => {
+    test("400 - responds with a 400 Bad Request if id is an invalid type", async () => {
+        const {body}  = await request(app).get("/api/articles/BadID/comments").expect(400)
+        expect(body.msg).toBe("Bad Request")
```

Comment:
Great test, potentially the error message could be a little bit more specific (especially as these are custom errors) but that's only a suggestion.